sourcegraph/about
# Redirect DPA to security dashboard

This closes #5771

### Changelog
- redirect `/dpa` and `/dpa.pdf` to our security dashboard

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure both `/dpa` and `/dpa.pdf` redirect to our security dashboard
